lib/codit/population/networks/city_config/city_cfg.py

```diff
@@ -14,6 +14,11 @@ def read_demographic_data(section, file_name, data_dir=share_dir() / "codit/data
 DEMO_DATA = (read_demographic_data('population', 'table-8.csv')["%"] / 100).to_dict()
 ROUNDING = 0.001  # rounding such that sum of rates equals 1
 
+
+POPULATION_CITY = 793139 # Total population estimate of Leeds (2019) Source: ONS* https://observatory.leeds.gov.uk/population/
```

Comment:
It's not hardcoded, I just used this Macro to save the size of population in this city_cfg.py, as it has been used in several places in the City_Overview.ipynb. I think it might be better to only change one place rather than several places. This number can be changed to any size. The only hardcoded data is the coordinates of the buildings saved in COORDINATES_CSV. These coordinates are real data extracted from OpenStreetMap. It can be any city, but the CITY_AREA has to be changed accordingly to that city's string to extract the coordinates of buildings in that city. 

e.g. Leeds' area string="area['ISO3166-2'='GB-LDS'][admin_level=8]"

Reply:
Don't want to hardcode, I just want to this Macro to save the size of population in this city_cfg.py, as it has been used in several places in the City_Overview.ipynb. I think it might be better to only change one place rather than several places. This number can be changed to any size. The only hardcoded data is the coordinates of the buildings saved in COORDINATES_CSV. These coordinates are real data extracted from OpenStreetMap. It can be any city, but the CITY_AREA has to be changed accordingly to that city's string to extract the coordinates of buildings in that city from households.ipynb.
e.g. Leeds' area string="area['ISO3166-2'='GB-LDS'][admin_level=8]"